.github/workflows/platform-backend-ci.yml

```diff
@@ -32,6 +32,13 @@ jobs:
         python-version: ["3.10"]
     runs-on: ubuntu-latest
 
+    services:
```

Comment:
Somehow this change makes the CI not kick in (also not failed either).
cc @Pwuts any clue?

Reply:
Okay it's working now.. Thanks!!